docs/shortcodes.md

```diff
@@ -6,6 +6,8 @@ Existing shortcodes can be converted to and from their block forms.
 
 They are held in a grouping called `WP Document Revisions`.
 
+Since the blocks make use of dynamically-generated cont, their contents are identical whether using a block or a shortcode/widget. 
```

Comment:
```suggestion
Since the blocks make use of dynamically-generated content, their contents are identical whether using a block or a shortcode/widget. 
```

Reply:
Thanks for that.  I'll reword it to

Since the blocks make use of dynamically-generated content, the same code is used to create this as for a shortcode/widget.

since the point is that the code used is the same.